README.md

```diff
@@ -97,6 +97,8 @@ root. Available configuration options:
  * `cache` default `false` - set to `true` to enable caching on Google Cloud using datastore
  * `debug` default `false` - set to `true` to log console messages from within the
  rendered pages.
+ * `renderOnly` - restrict the endpoint to only service requests for certain domains. Specified
```

Comment:
Clarify it's a prefix match including protocol (not domain). But also maybe it should be domain (e.g. users might miss http vs https).